AnnoMate/AppComponents/CNVPlotComponent.py

```diff
@@ -131,12 +131,14 @@ def gen_mut_scatter(maf_df, mut_sigma, sample):
     mut_scatter : go.Scatter
 
     """
+    cluster_assignment = maf_df.columns[maf_df.columns.isin(['Cluster_Assignment', 'cluster'])][0]
```

Comment:
I think we should pass `cluster_assignment_col` as a parameter in the function, rather than assuming the name is a "cluster". It can be propagated with `**kwargs` in `gen_cnv_plot` 

https://github.com/getzlab/AnnoMate/blob/bb3cb7e45d0fd6f2a41221b941fd686da872bd38/AnnoMate/AppComponents/CNVPlotComponent.py#L314C5-L314C17